static/bidder-info/generic.yaml

```diff
@@ -0,0 +1,154 @@
+          maintainer-email: DL-Zeta-SSP@zetaglobal.com
+          app-media-types:
+            - banner
+            - video
+          site-media-types:
+            - banner
+            - video
+          supported-vendors:
+          vendor-id: 833
+        usersync:
+          enabled: true
+          cookie-family-name: zeta_global_ssp
+          redirect:
+            url: https://ssp.disqus.com/redirectuser?sid=GET_SID_FROM_ZETA&gdpr={{gdpr}}&gdpr_consent={{gdpr_consent}}&us_privacy={{us_privacy}}&r={{redirect_url}}
+            uid-macro: 'BUYERUID'
+            support-cors: false
+      blue:
+        enabled: false
+        endpoint: https://prebid-us-east-1.getblue.io/?src=prebid
+        meta-info:
+          maintainer-email: prebid@getblue.io
+          site-media-types:
+            - banner
+          app-media-types:
+            - banner
+          supported-vendors:
+          vendor-id: 620
+      cwire:
+        enabled: false
+        endpoint: https://ortb.cwi.re/v1/bid
+        modifying-vast-xml-allowed: false
+        endpoint-compression: gzip
+        meta-info:
+          maintainer-email: devs@cwire.com
+          app-media-types:
+          site-media-types:
+            - banner
+          supported-vendors:
+          vendor-id: 1081
+      adsinteractive:
+        enabled: false
+        endpoint: http://bid.adsinteractive.com/prebid
+        modifying-vast-xml-allowed: false
+        meta-info:
+          maintainer-email: it@adsinteractive.com
+          app-media-types:
+            - banner
+          site-media-types:
+            - banner
+          supported-vendors:
+          vendor-id: 1212
+        usersync:
+          cookie-family-name: adsinteractive
+          redirect:
+            url: https://sync.adsinteractive.com/getuid?{{redirect_url}}&gpp={{gpp}}&gpp_sid={{gpp_sid}}
+            support-cors: false
+            uid-macro: '$AUID'
+      nativo:
+        enabled: false
+        endpoint: https://exchange.postrelease.com/esi?ntv_epid=7
+        pbs-enforces-ccpa: false
+        meta-info:
+          maintainer-email: prebiddev@nativo.com
+          app-media-types:
+            - banner
+            - video
+            - native
+          site-media-types:
+           - banner
+           - video
+           - native
+          supported-vendors:
+          vendor-id: 263
+        usersync:
+          cookie-family-name: nativo
+          redirect:
+            url: http://jadserve.postrelease.com/suid/101787?gdpr={{gdpr}}&gdpr_consent={{gdpr_consent}}&us_privacy={{us_privacy}}&ntv_gpp_consent={{gpp}}&ntv_r={{redirect_url}}
+            support-cors: false
+            uid-macro: 'NTV_USER_ID'
+    meta-info:
+      maintainer-email: maintainer@example.com
+      app-media-types:
+        - banner
+        - video
+        - native
+        - audio
+      site-media-types:
+        - banner
+        - video
+        - native
+        - audio
+      supported-vendors:
+      vendor-id: 0
```

Comment:
The PBS-Java config is not valid for PBS-Go. This should break out into one file per alias of the generic adapter. 

IMHO the root generic adapter is a special case which should not have a `generic.yaml` file, which likely requires extra logic.